### ISSUE TITLE
Blank entry

### ISSUE THREAD

**Issue by preke** [Alex-Fabbri/Multi-News#30]
In the source data "Raw data, bad retrievals removed"->"test.src.cleaned", line 4737 is blank

**Comment by Alex-Fabbri** [comment]
Good catch! That example should be removed/ignored in the next version of this data.
